### PR TITLE
A Tiny JS World

### PR DESCRIPTION
**Pull request by MosQuitO404**
### A Tiny JS World

[Demo](https://mosquito404.github.io/a-tiny-JS-world/) |
[Code base](https://github.com/MosQuitO404/a-tiny-JS-world)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by MosQuitO404** [comment]
Hi! I have completed self-checks and fixed code accordingly.

File: submissions/MosQuitO404/a-tiny-js-world/index.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+// ======== OUTPUT ========
+/* Use print(message) for output.
+   Default tag for message is <pre>. Use print(message,'div') to change containing element tag.
+
+   Message can contain HTML markup. You may also tweak index.html and/or styles.css.
+   However, please, REFRAIN from improving visuals at least until your code is reviewed
+   so code reviewers might focus on a single file that is index.js.
+   */
+
+   const inhabProp = ['species', 'name', 'gender', 'legs', 'hands', 'saying'];
```
Comment (OleksiyRudenko) [inline submissions/MosQuitO404/a-tiny-js-world/index.js:53]: https://kottans.org/documentation/docs/doc/code-review/#code-style

File: submissions/MosQuitO404/a-tiny-js-world/index.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+   print(inhabitants.map(obj => inhabProp.map(elem => obj[elem]).join('; ')).join('\n'));
```
Comment (OleksiyRudenko) [inline submissions/MosQuitO404/a-tiny-js-world/index.js:55]: https://kottans.org/documentation/docs/doc/code-review/#code-style

**Review by OleksiyRudenko — CHANGES_REQUESTED** [review]
@MosQuitO404 good job. Let's polish it.

File: submissions/MosQuitO404/a-tiny-js-world/index.js (hunk 1 above)
Comment (MosQuitO404) [reply submissions/MosQuitO404/a-tiny-js-world/index.js]: Hi! Resolved. Thanks!

File: submissions/MosQuitO404/a-tiny-js-world/index.js (hunk 2 above)
Comment (MosQuitO404) [reply submissions/MosQuitO404/a-tiny-js-world/index.js]: Resolved too :)

**Review by OleksiyRudenko — APPROVED** [review]
@MosQuitO404 well done!

**Pull request merged**